src/test/resources/application-integration.properties

```diff
@@ -0,0 +1 @@
+spring.flyway.enabled=true
```

Comment:
yes we use hocs-data to manage the flyway at the moment, but we need it to run for the tests..

Reply:
https://github.com/UKHomeOffice/hocs-info-service/pull/428/files/9a21f36ef080a8d66c4fa82a510e3097af0d1d3d#diff-0218b149293465d6bb4256c4c70e7d331ca030aaa1e56b1bcaf84f6f9a09b40eL1 its because we'd duplicated the application properties here but taken the disabled line away, so it now uses the real application properties we have to explicity enable it to override